exp/views/analytics.py

```diff
@@ -57,7 +57,7 @@ def get_context_data(self, **kwargs):
         # Responses for studies - only include real (non-preview) responses here
         annotated_responses = (
             get_annotated_responses_qs()
-            .filter(study__in=studies_for_user, is_preview=False)
+            .filter(study__in=studies_for_user, is_preview=False, child__deleted=False)
```

Comment:
Weirdly I think here we do want to include deleted children - for counting how many responses over time, demographics of people who participated, etc. we care about info entered at time of participation. It's just the separate display of currently-registered children (kids who might participate in future studies) where we don't want to include deleted.